pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go

```diff
@@ -748,6 +748,12 @@ func (r *Reconciler) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 			SkipIf:       o.Shoot.IsWorkerless || o.Shoot.HibernationEnabled,
 			Dependencies: flow.NewTaskIDs(deployWorker),
 		})
+		deployExtensionResourcesAfterWorker = g.Add(flow.Task{
+			Name: "Deploying extension resources after workers",
```

Comment:
We probably also want to switch messages for hibernation, similar to https://github.com/gardener/gardener/blob/4fb0fd24dbf1a645d31adeac02960cd6b09f99e4/pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go#L125-L132

Reply:
I thought this was mostly done because here the order is inverted between regular reconcile & hibernation. I am unsure what messages we'd have then except for changing `Deploying` to `Hibernation`.

Reply:
Ah, you're right. Then let's spare the effort.